src/pages/contact/Contact.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
 
+import "./contact.scss";
+import logo from './logo.png';
+import email from './email.png';
+import location from './location.png';
+
+
 export default function Contact() {
-  return <div></div>;
-}
+    return (
+        <div>
+            <div className='three-divs'>
+                <div className='contact-box'>
+                    <img src={logo} alt="phone" />
+                    <p className='phone'>Phone</p>
+                    <p className='call'>Call us: 0333 015 0000</p>
+                </div>
```

Comment:
you've got to create a component for "contact-box" and use this component here 3 times